tests/lib/snaps/test-snapd-success-exit-status/meta/snap.yaml

```
@@ -0,0 +1,15 @@
+name: test-snapd-success-exit-status
+version: 1.0
+summary: Test snap for success-exit-status functionality
+description: |
+  This snap contains a daemon configuration to test the success-exit-status
+  functionality in snapd. It includes a service that exit with different codes
+  and demonstrates how systemd should handle them based on the success-exit-status
+  configuration.
+
+apps:
+  test-snapd-success-exit-status:
+    command: bin/service-exit-code
+    daemon: simple
+    restart-condition: on-failure
+    success-exit-status: [42, 250]
```

the test data is often used as reference, perhaps we can tweak it so to provide a good example of how we expect the feature to be used in snaps:

```suggestion
    success-exit-status: 
      - 42   # magic number
      - 250 # nothing in particular
      - 235 # EXIT_CHOWN
```